.github/workflows/build_checksums.yml

```diff
@@ -26,9 +26,9 @@ jobs:
         set -x
         TAG_NAME=${{ github.event.inputs.tag }}
         if [ -z "$TAG_NAME" ]; then
-          url=https://api.github.com/repos/NREL/EnergyPlus/releases/latest
+          url=https://api.github.com/repos/$GITHUB_REPOSITORY_OWNER/EnergyPlus/releases/latest
         else
-          url=https://api.github.com/repos/NREL/EnergyPlus/releases/tags/$TAG_NAME
+          url=https://api.github.com/repos/$GITHUB_REPOSITORY_OWNER/EnergyPlus/releases/tags/$TAG_NAME
```

Comment:
That is it

Reply:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables

> GITHUB_REPOSITORY_OWNER	The repository owner's name. For example, octocat.

Reply:
:+1: I guess in an extreme case, someone could rename their repo?  But let's say that won't happen :) 

Thanks @jmarrec, merging this right in.

Reply:
Could use $GITHUB_REPOSITORY

```
          url=https://api.github.com/repos/$GITHUB_REPOSITORY/releases/latest
```

But really this PR is mostly just for me, and maybe one or two forked repos ever.